examples/llm_embedding.py

```diff
@@ -81,15 +85,38 @@ def __call__(self, sentences: List[str]) -> Tensor:
         return embeddings
 
 
+class VoyageaiEmbedding:
+    dimension: int = 4096
+    text_embedder_batch_size: int = 8
+
+    def __init__(self, model: str = 'voyage-01'):
+        # Please run `pip install voyageai` to install the package
+        self.model = model
+
+    def __call__(self, sentences: List[str]) -> Tensor:
+        import voyageai
+        voyageai.api_key = api_key
+        from voyageai import get_embeddings
+        items = get_embeddings(sentences, model=self.model)
```

Comment:
Just curious, what's the type of `items`? Can we have the same type annotation for the other API embeddings as well?

Reply:
`List[List[Float]]`